lib/src/app/navigation/route_page_manager.dart

```diff
@@ -0,0 +1,151 @@
+import 'package:allthenews/src/app/navigation/app_path.dart';
+import 'package:allthenews/src/app/navigation/app_router_information_parser.dart';
+import 'package:allthenews/src/ui/pages/authentication/login/login_page.dart';
+import 'package:allthenews/src/ui/pages/authentication/registration/registration_page.dart';
+import 'package:allthenews/src/ui/pages/dashboard/dashboard_page.dart';
+import 'package:allthenews/src/ui/pages/dashboard/news/latest/latest_news_page.dart';
+import 'package:allthenews/src/ui/pages/dashboard/news/most_popular/most_popular_news_page.dart';
+import 'package:allthenews/src/ui/pages/home/bottom_bar_notifier.dart';
+import 'package:allthenews/src/ui/pages/home/home_page.dart';
+import 'package:allthenews/src/ui/pages/location/location_page.dart';
+import 'package:allthenews/src/ui/pages/presentation/presentation_page.dart';
+import 'package:allthenews/src/ui/pages/profile/profile_page.dart';
+import 'package:allthenews/src/ui/pages/settings/settings_page.dart';
+import 'package:allthenews/src/ui/pages/web_view/web_view_page.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class RoutePageManager extends ChangeNotifier {
+  static RoutePageManager of(BuildContext context) =>
+      Provider.of<RoutePageManager>(context, listen: false);
+
+  final BottomBarNotifier _allTheNewsAppState = BottomBarNotifier();
```

Comment:
to nie powinno być raczej `_bottomBarState` lub  coś w tym stylu? dlaczego bottom bar miałby odpowiadać za cały  stan ekranów? co z ekranami poza bottom barem?

Reply:
ostała się tu nieprawidłowa nazwa sprzed którejś tam wcześniejszej koncepcji